ts/store/reducers/index.ts

```diff
@@ -31,7 +31,7 @@ import walletReducer from "./wallet";
 export const authenticationPersistConfig: PersistConfig = {
   key: "authentication",
   storage: createSecureStorage(),
-  blacklist: ["deepLink"]
+  blacklist: ["deferred-navigation"]
```

Comment:
is the `deferred-navigation` state stored in the `authentication` store? @fpersico

Reply:
Not really.